docs/android/deployment/overview.md

```diff
@@ -13,133 +13,55 @@ ms.date: 03/17/2022
+Making your app available for direct download is most useful for a controlled subset of users, such as an enterprise environment or an app that is only meant for a small or well-specified set of users. Server and email distribution are also simpler publishing models, requiring less preparation to publish an app, though apps may be blocked as an email attachment.
 
-Here's an example of an automatically generated manifest file with the package and version information specified:
+The Amazon Mobile App Distribution Program enables mobile app developers to distribute and sell their applications on Amazon. Users can discover and shop for apps on their Android-powered devices by using the Amazon App Store application.
 
-```xml
-<manifest
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:versionCode="1"
-    package="com.hi.companyname.myproject"
-    android:versionName="1.0.0">
-    <!-- other settings -->
-</manifest>
-```
+Google Play is arguably the most comprehensive and popular marketplace for Android applications. Google Play allows users to discover, download, rate, and pay for applications by clicking a single icon either on their device or on their computer. Google Play also provides tools to help in the analysis of sales and market trends and to control which devices and users may download an application.
 
-For more information about the manifest, see [Google Android App Manifest Overview](https://developer.android.com/guide/topics/manifest/manifest-intro).
+## Related Links
 
-## Create a keystore file
+This section links to articles that may help in publishing an app to an app store such as Google Play.
 
-Your app package should be signed. You use a keystore file to sign your package. The Java/Android SDKs includes the tools you need to generate a keystore. After generating a keystore file, you'll add it to your project and configure your project file to reference it. The Java SDK should be in your system path so that you can run the _keytool_ tool.
+<!--
+- [Build Process](~/android/deploy-test/building-apps/build-process.md)
+- [Linking](~/android/deploy-test/linker.md)
+- [Obtaining A Google Maps API Key](~/android/platform/maps-and-location/maps/obtaining-a-google-maps-api-key.md)
+- [Deploy via Visual Studio App Center](/appcenter/distribution/stores/googleplay)
+- [Application Signing](https://source.android.com/security/apksigning/)
+-->
 
-Perform the following steps to create a keystore file:
+### Google Play app store
 
-01. Open a terminal and navigate to the folder of your project.
+- [Publishing on Google Play](https://developer.android.com/distribute/googleplay/publish/index.html)
+- [Google Application Licensing](https://developer.android.com/guide/google/play/licensing/index.html)
+- [Android.Play.ExpansionLibrary](https://github.com/mattleibow/Android.Play.ExpansionLibrary)
```

Comment:
If you follow this link you'll see he's deprecated the library!

Reply:
```suggestion
```